packages/whats-new/src/components/page-control.js

```diff
@@ -0,0 +1,36 @@
+/**
```

Comment:
This looks good, did you see we also had a similar component in the packages? It was extracted with the Gutenberg pagination control in mind.

`packages/components/src/pagination-control`

example in
`packages/tour-kit/src/variants/wpcom/components/wpcom-tour-kit-step-card-navigation.tsx`

Reply:
I wasn't aware of it. I wired it up and it works pretty well.
Will change the code to use this. Thanks!